pct.sh

```diff
@@ -17,7 +17,18 @@ if [[ -v EXTRA_MAVEN_PROPERTIES ]]; then
 	MAVEN_PROPERTIES="${MAVEN_PROPERTIES}:${EXTRA_MAVEN_PROPERTIES}"
 fi
 
-java -jar pct.jar \
+#
+# Plugin Compatibility Tester (PCT) requires custom --add-opens directives when running on Java 17.
+# As a temporary workaround, we pass in these directives when invoking PCT. When
+# jenkinsci/plugin-compat-tester#352 is merged and released, and when this repository has upgraded
+# to that release, this workaround can be deleted.
+#
+java \
+	--add-opens java.base/java.lang.reflect=ALL-UNNAMED \
```

Comment:
Means that tests are no longer possible using JDK 8. Finally the impetus for me to run

```bash
sudo update-java-alternatives -s java-1.11.0-openjdk-amd64
```

:grin:

Reply:
A timely merge and release of jenkinsci/plugin-compat-tester#352 would prevent that from being a problem.